tuxemon/states/combat/combat.py

```diff
@@ -473,18 +473,23 @@ def transition_phase(self, phase: CombatPhase) -> None:
             self.sort_action_queue()
 
         elif phase == "post action phase":
+            # remove actions from fainted users from the pending queue
+            self._pending_queue = [
+                pend
+                for pend in self._pending_queue
+                if pend.user
+                and isinstance(pend.user, Monster)
+                and not fainted(pend.user)
+                or fainted(pend.target)
```

Comment:
need to think about this

Reply:
please explain the filtering here

Reply:
Is this meant to be `not (fainted(pend.user) or fainted(pend.target))` ?

I think you might want one of the following, don't mind which one.

```
 self._pending_queue = [
                pend
                for pend in self._pending_queue
                if pend.user
                and isinstance(pend.user, Monster)
                and not fainted(pend.user)
                and not fainted(pend.target)
```

```
 self._pending_queue = [
                pend
                for pend in self._pending_queue
                if pend.user
                and isinstance(pend.user, Monster)
                and not (fainted(pend.user) or fainted(pend.target))
```

Reply:
No, pend.target is fine, it was only the pend.user the issue, thank I forgot a "not"
```
class EnqueuedAction(NamedTuple):
    user: Union[Monster, NPC, None]
    method: Union[Technique, Item, Condition, None]
    target: Monster
```